src/toil_scripts/rnaseq_cgl/test/test_rnaseq_cgl.py

```diff
@@ -2,27 +2,30 @@
 import subprocess
 import shutil
 import textwrap
+from uuid import uuid4
 
 from boto.s3.connection import S3Connection, Bucket, Key
 
 
 def test_rnaseq_cgl(tmpdir):
-    work_dir = str(tmpdir)
-    create_config_and_manifest(work_dir)
+    tmp_dir = str(tmpdir)
```

Comment:
There are several throwaway directories floating around here (`tmpdir`, `tmp_dir`, and `work_dir`). Can you document what each one of them does or how they are different?